save_princess_challenge/lib/display_path.rb

```diff
@@ -1,15 +1,54 @@
 #!/bin/ruby
+class DisplayPath
+  def initialize; end
 
-def displayPathtoPrincess(n,grid)
-  return ""
-end
+  def displayPathtoPrincess(n,grid)
+    m_y_coord, m_x_coord = find_center(n)
+    p_y_coord, p_x_coord = find_princess(n, grid)
+    moves = []
+    until m_x_coord == p_x_coord && m_y_coord == p_y_coord
+      if p_y_coord == 0
+        m_y_coord -= 1
+        moves.push 'UP'
+      else
+        m_y_coord += 1
+        moves.push 'DOWN'
+      end
+      if p_x_coord == 0
+        m_x_coord -= 1
+        moves.push 'LEFT'
+      else
+        m_x_coord += 1
+        moves.push 'RIGHT'
+      end
+    end
+    p moves.join('\n')
```

Comment:
Using this array to test the output in Minitest, hackerrank is checking standard output. I should find out ways to test that.